addons/project_purchase/models/project_project.py

```diff
@@ -153,35 +153,53 @@ def _get_profitability_items(self, with_action=True):
             )
             if invoice_lines:
                 amount_invoiced = amount_to_invoice = 0.0
+                amount_billed = amount_to_bill = 0.0
```

Comment:
If we're defining multiple variables in one go, why not all of them in one line?